wdl2cwl/WdlV1_1ParserVisitor.py

```diff
@@ -356,18 +354,17 @@ def visitMeta_kv(self, ctx:WdlV1_1Parser.Meta_kvContext):
 
     # Visit a parse tree produced by WdlV1_1Parser#parameter_meta.
     def visitParameter_meta(self, ctx:WdlV1_1Parser.Parameter_metaContext):
-        return self.visitChildren(ctx)
+        self.task_parameter_meta_check = 1
 
 
     # Visit a parse tree produced by WdlV1_1Parser#meta.
     def visitMeta(self, ctx:WdlV1_1Parser.MetaContext):
-        return self.visitChildren(ctx)
+        self.task_meta_check = 1
```

Comment:
This line is never called by any of the tests. Can you either add a meta section to one of the tests, or remove this?